### DISCUSSION TITLE
How to put my data to GPU

### DISCUSSION THREAD

**Discussion by HtwoOtwo** [open-mmlab/mmcv#2412] [Q&A]
dataset = build_dataset(self.cfg.data.test)
data_loader = build_dataloader(
    dataset,
    samples_per_gpu=self.cfg.data.samples_per_gpu,
    workers_per_gpu=self.cfg.data.workers_per_gpu,
    shuffle=False,
    round_up=True)
result = single_gpu_test(self.infer_model, data_loader)
-----------------------------------------------------------------
e.g.  above code, How to put my data to GPU

**Comment by zhouzaida** [comment]
Hi, data will be copied to GPU when calling the `forward` method of `self.infer_model` which was wrapped by `MMDP` or `MMDDP`.
